src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,16 +1,20 @@
 package mate.academy.service;
 
 public class AuthenticationService {
+    private final UserService userService = new UserService();
     /**
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
      * @param email - user's email
      * @param password - user's password
+     *                 new User("bob@i.ua", "1234"),
+     *             new User("alice@i.ua", "1234")
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+
     public boolean login(String email, String password) {
-        return false;
+        return userService.findByEmail(email).getPassword().equals(password);
```

Comment:
this can throw NPE -- you should still check for null :)  You should have one line to get user from service, and one line to return an expression (use && there)